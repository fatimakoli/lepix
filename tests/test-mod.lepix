fun mod(num:int, denom:int): int {
	var multiplier:int = num/denom;
	return num - denom * multiplier;
}

fun main(): int {
	print(mod(32, 6));
	print(mod(40, 8));
	print(mod(46, 10));
	print(mod(57, 25));
	print(mod(34, 6));
	print(mod(97, 5));
	print(mod(17, 10));
	return 0;
}