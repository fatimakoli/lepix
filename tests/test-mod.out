2
0
6
7
4
2
7
